anaUltraLatency.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/env python
+#! /usr/bin/env python
```

Comment:
What is the rationale for this change (and in subsequent files: `anaUltraScurve.py`, `anaUltraThreshold.py`, `anaXDAQLatency.py`, `ana_scans.py`, and `anautilities.py`)?

Are you sure this change, while possibly motivated for making `Travis CI` work, will not impact performance on production machines? Specifically the P5 machines?

Reply:
Debian/Ubuntu don't have `/bin/env` in any supported version. Performance won't be affected at all, `env` is just a tool that searches `$PATH` and standard locations for an executable and launches it. The only possible problem would be `SL` to remove `/usr/bin/env` in a future version, but IIRC there is some standard for Linux distros that says `/usr/bin` should be the default for executables.